collected/icla/markewaite/committer.properties

```diff
@@ -0,0 +1,4 @@
+name=Mark Waite
+email=mark.earl.waite@github.com
```

Comment:
I'm not sure I believe you have an   `@github.com` mailing address @MarkEWaite  :smiling_imp:

Reply:
Oops.  Fixed and force pushed the fix (to hide my shame...)